.github/workflows/ci-core-reusable.yml

```diff
@@ -33,7 +33,6 @@ jobs:
           ci_run zk db setup
           ci_run zk compiler all
           ci_run zk contract build
-          ci_run zk run yarn
```

Comment:
Why is it removed? I'm fine with it If it works, just curious

Reply:
@perekopskiy https://github.com/matter-labs/zksync-era/pull/296/files#diff-293b35dbbaf364cec87320ca4091f93d974165e35cc640d7f44cfb9f51ebed0dL32 there's a duplicate line above